lms/static/js/student_account/views/account_settings_fields.js

```diff
@@ -48,48 +49,48 @@
 
         AccountSettingsFieldViews.PasswordFieldView = FieldViews.LinkFieldView.extend({
 
-            initialize: function (options) {
+            initialize: function(options) {
                 this._super(options);
                 _.bindAll(this, 'resetPassword');
             },
 
-            linkClicked: function (event) {
+            linkClicked: function(event) {
                 event.preventDefault();
                 this.resetPassword(event);
             },
 
-            resetPassword: function () {
-                var data = {};
+            resetPassword: function() {
+                var data = {},
+                    view = this;
                 data[this.options.emailAttribute] = this.model.get(this.options.emailAttribute);
-
-                var view = this;
                 $.ajax({
                     type: 'POST',
                     url: view.options.linkHref,
                     data: data,
-                    success: function () {
+                    success: function() {
                         view.showSuccessMessage();
                     },
-                    error: function (xhr) {
+                    error: function(xhr) {
                         view.showErrorMessage(xhr);
                     }
                 });
             },
 
-            successMessage: function () {
-                return this.indicators.success + interpolate_text(
+            successMessage: function() {
+                return HtmlUtils.interpolateHtml(
                     gettext(
-                        /* jshint maxlen: false */
-                        'We\'ve sent a message to {email_address}. Click the link in the message to reset your password.'
+                        '{success_indicator}We\'ve sent a message to {email_address}. Click the link in the message to reset your password.'  // jshint ignore:line
```

Comment:
Opinion: I'd rather see double-quote used so the inner quote doesn't need to be escaped.